annotation-server/Dockerfile

```diff
@@ -1,24 +1,26 @@
+# Note: this docker file is meant to be run from the project root as we need to
+# add local modules You can build this image by running `docker build -t
+# annotation-server -f ./annotation-server/Dockerfile .` from the project root.
 FROM node:16
 
-RUN apt-get update || : && apt-get install python -y
-RUN apt-get install python3-pip -y
-
-# Create app directory
 WORKDIR /usr/src/app
 
-# Install app dependencies
-COPY package.json ./
-COPY yarn.lock ./
-COPY requirements.txt ./
+RUN apt-get update || : && apt-get install python -y
+RUN apt-get install python3-pip -y
 
-RUN yarn
-RUN pip3 install -r requirements.txt
+# Copy top-level package json (needed to register workspaces)
+COPY ./package.json .
+COPY ./yarn.lock .
 
-# Bundle app source
-COPY . .
+# Copy common module
+COPY ./common ./common
 
-RUN yarn build
+# Copy annotation server
+COPY ./annotation-server ./annotation-server
```

Comment:
We should probably not copy the entire folder here rather `package.json`, `yarn.lock` and `requirements.txt`

Reply:
Docker-files have been optimized now. (from ca 1.8 GB to ~400MB)! Thank you for pointing this out.